src/udev/udev-rules.c

```diff
@@ -1177,7 +1177,7 @@ static void check_tokens_order(UdevRuleLine *rule_line) {
                 if (t->type == TK_M_RESULT)
                         has_result = true;
                 else if (has_result && t->type == TK_M_PROGRAM) {
-                        log_line_warning(rule_line, "Reordering RESULT check after PROGRAM assignment.");
+                        log_line_notice(rule_line, "style: reordering RESULT check after PROGRAM assignment.");
```

Comment:
Unlike commas and whitespace issues, this might be an indication of a real bug, especially if there are several PROGRAM assignments in the rule.  I'd suggest to keep this as a regular non-style warning.